frontend/src/components/BedManagement/BedManagement.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { useParams } from "react-router-dom";
+import { Table, Button } from "react-bootstrap";
+import {
+  getBedsForWard,
+  updateBedState
+} from "../../Services/BedManagementService";
+
+// Types
+import { Ward, Bed, BedState } from "../../datatypes/ListView.types";
+
+// Styles
+import "primereact/resources/themes/nova-light/theme.css";
+import "primereact/resources/primereact.min.css";
+import "primeicons/primeicons.css";
+import "./BedManagement.scss";
+
+const BedManagement: React.FunctionComponent = () => {
+  const { wardId } = useParams();
+  const [wardData, setWardData] = React.useState<Ward | null>(null);
+
+  React.useEffect(() => {
+    (async () => {
+      if (!wardId) {
+        throw new Error("Expected a ward id");
+      }
+      const ward = await getBedsForWard(wardId);
+      setWardData(ward);
+    })();
+  }, []);
+
+  function updateBedStateHelper(bedId: string, bedIsFree: boolean) {
+    if (bedIsFree) {
+      updateBedState(bedId, BedState.occupied);
+    } else {
+      updateBedState(bedId, BedState.free);
+    }
+  }
+
+  return (
+    <div>
+      {wardData && (
+        <div className="bed-management">
+          <h1>{wardData.name}</h1>
+
+          <Table bordered className="">
+            <thead>
+              <tr>
+                <th>Bettenplatz</th>
+                <th>Bettentyp</th>
+                <th>Aktion</th>
+              </tr>
+            </thead>
+            <tbody>
+              {wardData.beds.map((bed: Bed) => {
+                const bedIsFree =
+                  (bed.bedState as any) === BedState[BedState.free];
+
+                return (
+                  <tr key={bed.id} className={`--${bed.bedState}`}>
+                    <td>{bed.name}</td>
+                    <td>{bed.bedType}</td>
+                    <td>
+                      <Button
```

Comment:
this is only for changing the bed state to `free` or `occupied`, so i think a button would fit the purpose better

Reply:
Okay, then we should add an input somewhere else for the remaining states, but we can defer this to an issue easily 😉

Reply:
yep, you are right. i think it would fit the edit page of wards where you can add / edit / delete beds. this component here is dedicated for a quick overview of free / available beds and toggle the status of the beds, i think changing the type of a bed is not necessary in this component. but again, i am very open for a feature like that, if it makes senses from an UX point of view :)